internal/syscallfs/syscallfs_test.go

```diff
@@ -257,3 +260,100 @@ func (f *testFSAdapter) Open(name string) (fs.File, error) {
 func requireErrno(t *testing.T, expected syscall.Errno, actual error) {
 	require.True(t, errors.Is(actual, expected), "expected %v, but was %v", expected, actual)
 }
+
+var (
+	//go:embed testdata/wazero.txt
+	readerAtFS   embed.FS
+	readerAtFile = "wazero.txt"
```

Comment:
maybe better add empty file as well so that we won't crash when reading on the empty file?

Reply:
sure!